report/clients/standalone.js

```diff
@@ -29,8 +27,12 @@ function __initLighthouseReport__() {
   document.body.append(reportRootEl);
 
   document.addEventListener('lh-analytics', /** @param {Event} e */ e => {
-    // @ts-expect-error
-    if (window.ga) ga(e.detail.cmd, e.detail.fields);
+    const ce = /** @type {CustomEvent<{name: string, data?: {}}>} */(e);
+
+    if ('gtag' in window) {
+      // @ts-expect-error
+      window.gtag('custom_event', ce.detail.name, ce.detail.data ?? {});
```

Comment:
```suggestion
      window.gtag('event', ce.detail.name, ce.detail.data ?? {});
```

right?  https://developers.google.com/analytics/devguides/collection/ga4/events?client_type=gtag&sjid=15586495081079579696-NC#google_tag_gtagjs_overview